Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -1813,6 +1813,195 @@ def test_module(client: Client, *_) -> Tuple[str, Dict[Any, Any], Dict[Any, Any]
+        arg=ticket.get('sys_updated_on'),
+        arg_name='sys_updated_on',
+        required=False
+    )
+
+    demisto.debug(f'ticket_last_update is {ticket_last_update}')
+
+    if last_update > ticket_last_update:
+        demisto.debug(f'Nothing new in the ticket')
+        ticket = {}
+    else:
+        demisto.debug(f'ticket is updated: {ticket}')
+
+    # get latest comments
+    entries = []
+
+    sys_param_limit = args.get('limit', client.sys_param_limit)
+    sys_param_offset = args.get('offset', client.sys_param_offset)
+
+    sys_param_query = f'element_id={ticket_id}^element=comments^ORelement=work_notes'
+
+    comments_result = client.query('sys_journal_field', sys_param_limit, sys_param_offset, sys_param_query)
+    demisto.debug(f'Comments result is {comments_result}')
+
+    if not comments_result or 'result' not in comments_result:
+        demisto.debug(f'Pull result is {ticket}')
+        return [ticket]
+
+    for note in comments_result.get('result'):
+        entry_time = arg_to_timestamp(
+            arg=note.get('sys_created_on'),
+            arg_name='sys_created_on',
+            required=False
+        )
+        demisto.debug(f'entry_time is {entry_time}')
+
+        if last_update > entry_time:
+            continue
+
+        entries.append({
+            'Type': 1,
+            'Category': 'chat',
+            'ContentsFormat': 'text',
+            'Contents': note.get('value'),
+        })
+
+    original_ticket = result['result']
+
+    if original_ticket.get('resolved_by') != '':
+        demisto.info(f'ticket is closed: {original_ticket}')
+        entries.append({
+            'Type': EntryType.NOTE,
+            'Contents': {
+                'dbotIncidentClose': True,
+                'closeReason': 'From ServiceNow'
+            },
+            'ContentsFormat': EntryFormat.JSON
+        })
+
+    demisto.debug(f'Pull result is {ticket} + {entries}')
+
+    return [ticket] + entries
+
+
+def update_remote_system_command(client: Client, args: Dict[str, Any]) -> str:
+    """update-remote-system command: pushes local changes to the remote system
+
+    :type client: ``Client``
+    :param Client: XSOAR client to use
+
+    :type args: ``Dict[str, Any]``
+    :param args:
+        all command arguments, usually passed from ``demisto.args()``.
+        ``args['data']`` the data to send to the remote system
+        ``args['entries']`` the entries to send to the remote system
+        ``args['incidentChanged']`` boolean telling us if the local incident indeed changed or not
+        ``args['remoteId']`` the remote incident id
+
+    :return:
+        ``str`` containing the remote incident id - really important if the incident is newly created remotely
+
+    :rtype: ``str``
+    """
+    data = args.get('data')
+    entries = args.get('entries')
+    incident_changed = args.get('incidentChanged')
+    ticket_id = args.get('remoteId')
+    org_status = args.get('status')
```

Comment:
you can also use `delta` object which contains only the changed fields to be on the safe side(If the SNOW API support updating just part of the incident)